cdap-docs/admin-manual/source/operations/tx-maintenance.rst

```diff
@@ -69,25 +69,27 @@ To prune the invalid list manually, follow these steps:
 #. The final ``pruneTime`` is the minimum ``pruneTime`` across all replicated clusters (if
    any); let this be time ``t``.
 
-Now, the invalid transaction list can be safely pruned
-until ``(t - 1 day)`` using :ref:`a call <http-restful-api-transactions-truncate>`
-with the :ref:`Transaction Service HTTP RESTful API <http-restful-api-transactions-truncate>`.
+Now, the invalid transaction list can be safely pruned until ``(t - 1 day)`` using a call
+to :ref:`truncate invalid transactions before a specific time
+<http-restful-api-transactions-truncate>` with the :ref:`Transaction Service HTTP RESTful
+API <http-restful-api-transactions-truncate>`.
 
-The current length of the invalid transaction list can be obtained using 
-:ref:`a call <http-restful-api-transactions-number>` 
-with the :ref:`Transaction Service HTTP RESTful API <http-restful-api-transactions-number>`.
+The current length of the invalid transaction list can be obtained using a call to
+:ref:`retrieve the number of invalid transactions <http-restful-api-transactions-number>` 
+with the :ref:`Transaction Service HTTP RESTful API
+<http-restful-api-transactions-number>`.
 
 .. _tx-maintenance-pruning-replicated:
 
 Pruning Invalid Transactions in a Replicated Cluster
 ----------------------------------------------------
-- Copy over the latest transaction snapshots from the master cluster to the slave cluster.
+1. Copy over the latest transaction snapshots from the master cluster to the slave cluster.
 
-- Wait for a few minutes for the latest transaction state to be reloaded from the
-  snapshot.
+#. Wait for a few minutes for the latest transaction state to be reloaded from the
+   snapshot.
 
-- Run steps 1 to 5 from the above list on the slave cluster to find the ``pruneTime`` for
-  that slave cluster.
+#. Run steps 1 to 5 from the above list on the slave cluster to find the ``pruneTime`` for
```

Comment:
Now that I read this again, it would be good to say `Run steps 1 to 5 from the above section "Pruning Invalid Transactions"...`